samples/nfc/tnep_tag/src/main.c

```diff
@@ -14,10 +14,6 @@
 #include <nfc/tnep/tag.h>
 #include <nfc/ndef/msg.h>
 #include <nfc/ndef/text_rec.h>
-#include <logging/log.h>
-
-
-LOG_MODULE_REGISTER(dupa);
```

Comment:
Farewell, funny line!

Reply:
so sad :(

Reply:
device under permanent association... what's wrong with that!?

Reply:
:(